src/build.ts

```diff
@@ -4,23 +4,24 @@ import { IOType } from 'child_process';
 import { join } from 'path';
 
 import { runCommand as runCleanCommand } from '#commands/clean';
-import { runCommand as runStableCompileCommand } from '#commands/compile/index';
-import { runCommand as runGenerateTypesCommand } from '#commands/generate_types/index';
-import { runCommand as runInstallDfxExtensionCommand } from '#commands/install_dfx_extension';
+import { runCommand as runStableBuildCommand } from '#commands/compile/index';
```

Comment:
The `compile` directory needs to change to `build`

Reply:
Done